src/components/LastSearches/LastSearches.jsx

```diff
@@ -0,0 +1,48 @@
+import { useEffect, useState } from 'react';
+
+import clock from '@assets/lastSearches/clock.svg';
+import { getCookie, removeCookie } from '@utils/cookies';
+
+import styles from './LastSearches.module.css';
+
+export default function LastSearches() {
+    const [lastSearches, setLastSearches] = useState([]);
+
+    useEffect(() => {
+        const lastSearchesJson = getCookie('lastSearches');
+        const lastSearches = lastSearchesJson ? JSON.parse(lastSearchesJson) : [];
+        setLastSearches(lastSearches);
+    }, []);
+
+    const handleClear = () => {
+        removeCookie('lastSearches');
+        setLastSearches([]);
+    };
+
+    return (
+        <div className={styles['latestSearchesContainer']}>
+            <div className={`${styles['titles']} ${styles['latestSearchesTitles']}`}>
+                <h1>Latest searches</h1>
```

Comment:
This one shouldn't be there

Reply:
what do you mean??

Reply:
the sidebar is already merged into main

Reply:
yes! but not when ive made the branch. tomorrow ill see if i can make a rebase of main to this branch.